### PR TITLE
Feature create database

### PR DESCRIPTION
**Pull request by belalgamal46**
# **Create database**

- [x] Create animals table schema 
- [x] Create Insert commands to insert animals 
- [x] Create queries to read data in the table 
- [x] Update the README.md file and add the license file

**Find all animals whose name ends with "mon".**

Screenshot from 2023-01-17 15-26-53

**List the name of all animals born between 2016 and 2019.**

Screenshot from 2023-01-17 15-27-29

**List the name of all animals that are neutered and have less than 3 escape attempts.**

Screenshot from 2023-01-17 15-27-49

**List the date of birth of all animals named either "Agumon" or "Pikachu".**

Screenshot from 2023-01-17 15-28-12

**List name and escape attempts of animals that weigh more than 10.5kg**

Screenshot from 2023-01-17 15-28-34

**Find all animals that are neutered.**

Screenshot from 2023-01-17 15-28-51

**Find all animals not named Gabumon.**

Screenshot from 2023-01-17 15-29-02

**Find all animals with a weight between 10.4kg and 17.3kg**

Screenshot from 2023-01-17 15-29-19

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,5 +1,10 @@
...
 CREATE TABLE animals (
-    name varchar(100)
-);
+  id INT,
+  name VARCHAR(250),
+  date_of_birth date,
+  escape_attempts INT,
+  neutered boolean,
+  weight_kg decimal
+  );
```
Comment (Abenezer-Tilahun) [inline schema.sql]: - [ Optional ] Kindly I recommend you make the id column not null and auto incremented and to mark the id column as primary key.
`id int NOT NULL IDENTITY(1,1), PRIMARY KEY (id)`

File: queries.sql [hunk 2]
```diff
@@ -1,3 +1,15 @@
...
+SELECT date_of_birth  from animals WHERE name = 'Agumon' or name = 'Pikachu';
+SELECT name, escape_attempts  from animals WHERE weight_kg > 10.5;
+SELECT * from animals WHERE neutered = 't';
+SELECT * from animals WHERE name != 'Gabumon';
+SELECT * from animals WHERE weight_kg >= 10.4 and weight_kg <= 17.3;
+
+
+
+
+
```
Comment (Abenezer-Tilahun) [inline queries.sql]: - [ Optional ] Kindly I recommend you  to remove the empty space from your code base , because it make your code base unclear.

**Review by Abenezer-Tilahun — CHANGES_REQUESTED** [review]
Hi @belalgamal46  , 

Great job so far, There are some changes that need to be implemented in order to proceed to the next project 👍

## Highlights 👏🏻

✔️ No Linters error
✔️ You used correct Git -Flow.
✔️ Professional  README
✔️ Good commit messages

almost

## Required Changes ♻️
- [x] As stated the project requirement Kindly **add the screenshot with the results of your queries to your Pull Request description.**.

Capture

_Check the comments below the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by elmar8287 — APPROVED** [review]
Hi, @belalgamal46.

# APPROVED  👍 🥇 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 

Congratulations! 🎉 

### Highlights

- All suggestions are implemented! 👍 

### Optional suggestions

- N/A

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
### Please, remember to tag me (@elmar8287) in your question so I can receive the notification 👍 
giphy

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**